CRM/Pspsepa/Plugins/AdyenPspRunner.php

```diff
@@ -56,6 +56,7 @@ public function processRecord($record, $params) {
     $request->sendRequest();
     $response = json_decode($request->getResponseBody(), TRUE);
     $response_code = $request->getResponseCode();
+    CRM_Core_Error::debug_log_message("Received Adyen Response for Contribution {$contribution_id}: HTTP {$response_code}: {$request->getResponseBody()}");
```

Comment:
Logging to the error log should be made configurable, we usually use a constant for (de-)activating logging, like this:
```PHP
if (defined('PSPSEPA_LOGGING') && PSPSEPA_LOGGING) {
    CRM_Core_Error::debug_log_message('...');
}
```
This constant can then be defined within the civicrm.settings.php file.